README.md

```diff
@@ -1476,7 +1476,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
    * [wakatime.com](https://wakatime.com/) — Quantified self-metrics about your coding activity, using text editor plugins, limited plan for free.
    * [WebComponents.dev](https://webcomponents.dev/) — In-browser IDE to code web components in isolation with 58 templates available, supporting stories and tests.
    * [PHPSandbox](https://phpsandbox.io/) — Online developement environment for PHP.
-
+   * [PHPize.online](https://phpize.online/) — Free online environment for experimenting with and sharing PHP and SQL code (PDO ($pdo), mysqli ($mysqli) & Laravel query builder ($db)). The plattform supports last PHP versions (5.6 - 8.2) and most popular RDBMS such as MySQL (5.7, 8.0), MariaDB 10, SQLite 3, PostgreSQL (11 - 15), MS SQL Server (2017, 2019, 2022), Oracle Database (19c, 21c) 
```

Comment:
please keep the new line below.

to me this is a bit minimal to be considered for the list, every language has a play ground its not really a service.  Contrast with https://phpsandbox.io/ thats a full, modern, IDE.  I think this is a bit old and limited.

Reply:
New line added. 
The PHPize is bit more then regular playground, it give ability to play with PHP and SQL with in same browser window. I developed it for support my Stackoverflow answers by online live code examples

Reply:
I understand, but I think the list has a more broader interest so I think it's not a great fit